### PR TITLE
[feature/speaker_name_theme]: change speaker name text style

### LINKED ISSUE #828: Adjust Text Style of Speakers in Timetable to Match Figma Design
**Idea Description**
The text style of the speakers in the timetable needs to be adjusted to match the design specified in Figma. 

**Reference images and links**
https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI?node-id=54462%3A36316&mode=dev
image

### PR DESCRIPTION
**Pull request by KwakEuiJin**
## Issue
- close #828

## Overview (Required)
- TimeTableListItem.kt : Change speaker name Text style 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableListItem.kt [hunk 1]
```diff
@@ -141,8 +141,7 @@ fun TimetableListItem(
                     Spacer(modifier = Modifier.size(10.dp))
                     Text(
                         text = speaker.name,
-                        fontSize = 14.sp,
-                        fontWeight = FontWeight.Medium,
+                        style = typography().bodyMedium,
```
Comment (takahirom) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableListItem.kt:144]: We can use MaterialTheme.typography here 👀

Comment (KwakEuiJin) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableListItem.kt:144]: oh i fixed this
thanks😁

Comment (KwakEuiJin) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableListItem.kt]: oh sorry I'm done pushing now !

**Comment by takahirom** [comment]
Please fix the conflict 🙏

**Comment by KwakEuiJin** [comment]
@takahirom 
Fixed conflict 
thanks !

**Review by takahirom — APPROVED** [review]
Looks great! Thanks

**Pull request merged**